docs/src/main/sphinx/functions/bitwise.rst

```diff
@@ -12,22 +12,42 @@ Bitwise functions
         SELECT bit_count(-7, 64); -- 62
         SELECT bit_count(-7, 8); -- 6
 
+.. function:: bitwise_not(x) -> bigint
```

Comment:
Why are you reordering the functions to be not alphabetically sorted?

Reply:
Sorry, my bad. I wasn't aware of the alphabetic ordering. My thinking was, that the logical operations (and, not, or and xor) could be sorted according to their arity (not - one argument, and - two arguments, or - two arguments, xor - two argument.)

I will revert this part.